src/pubsub/examples/src/topic/create_topic.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START pubsub_create_topic]
+use google_cloud_pubsub::{client::TopicAdmin, model::Topic};
+
+pub async fn sample(client: &TopicAdmin, topic_name: &str) -> anyhow::Result<()> {
+    let topic: Topic = client.create_topic().set_name(topic_name).send().await?;
```

Comment:
What is the motivation to use a type annotation here? And should we make it easier to grok what is the structure of a topic name?


```suggestion
    let topic = client.create_topic().set_name(format!("projects/{project_id}/topics/{topic_id}")).send().await?;
```